webpack/assets/javascripts/react_app/components/ForemanModal/ForemanModalUsage.stories.js

```diff
@@ -255,3 +255,45 @@ export const asConfirmationDialog = () =>
 asConfirmationDialog.story = {
   name: 'As a confirmation dialog',
 };
+
+export const asConfirmationDialogWithCustomButtons = () =>
+  React.createElement(() => {
+    const dispatch = useDispatch();
+    return (
+      <Story>
+        <Button
+          bsStyle="primary"
+          onClick={() =>
+            dispatch(setModalOpenAction({ id: 'confirmationButtonsModal' }))
+          }
+        >
+          Show Modal
+        </Button>
+        <ForemanModal
+          id="confirmationButtonsModal"
+          title="With customized button text"
+          submitProps={{
+            url: '/api/question',
+            message: 'Thank you for your input',
+            method: 'post',
+            onSuccess: () => {},
+            submitBtnProps: {
+              btnText: 'Agree',
+              bsStyle: 'default',
+            },
+            cancelBtnProps: {
+              btnText: 'Disagree',
+              bsStyle: 'danger',
+            },
+          }}
+        >
+          One does not simply walk into Mordor.
+          <ForemanModal.Footer />
+        </ForemanModal>
+      </Story>
+    );
+  });
+
+asConfirmationDialogWithCustomButtons.story = {
+  name: 'As confirmation dialog with customized buttons',
+};
```

Comment:
The `story: { name: ... }` object is not required if you don't need to change the name or have special characters.  Storybook will transform `asConfirmationDialogWithCustomButtons` to "As confirmation dialog with custom buttons".  That said, if you want to change that word from "custom" to "customized" you could leave it as is, or change the function name.